backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -202,7 +215,14 @@ public StreetAddress getAddress() {
   }
 
   public String getTelephone() {
-    return telephone;
+    if (primaryPhone == null) {
+      return "";
+    }
+    return primaryPhone.getNumber();
+  }
+
+  public List<PhoneNumber> getPhoneNumbers() {
```

Comment:
maybe i'm missing something, where does phoneNumbers get set? don't see it in the constructor

Reply:
They're set through the `Person` service: https://github.com/CDCgov/prime-simplereport/blob/b449fde556b64f392ccbd272112cca837af90a17/backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java#L270-L292

The `person` associated with a phone number has to be fully saved before any can be added to avoid `transient instance`-esque errors.

Reply:
This is probably better addressed by making the relationship not bi-directional in hibernate—the PhoneNumber object doesn't actually need to know (at the java level) who it belongs to.

Reply:
https://en.wikibooks.org/wiki/Java_Persistence/ElementCollection actually uses phone numbers as the example case for using ElementCollection instead of ManyToOne. 😁